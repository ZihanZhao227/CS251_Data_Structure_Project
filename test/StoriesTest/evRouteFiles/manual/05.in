30
0 0 0 CITY Start
1 175 120 CHARGING_STATION CHARGING_STATION_1
2 122 72 CHARGING_STATION CHARGING_STATION_2
3 141 114 RESTAURANT RESTAURANT_3
4 132 119 CHARGING_STATION CHARGING_STATION_4
5 61 20 REGULAR_WAYPOINT REGULAR_WAYPOINT_5
6 3 115 CHARGING_STATION CHARGING_STATION_6
7 138 99 CHARGING_STATION CHARGING_STATION_7
8 180 13 CHARGING_STATION CHARGING_STATION_8
9 152 63 CHARGING_STATION CHARGING_STATION_9
10 94 42 CHARGING_STATION CHARGING_STATION_10
11 140 189 CHARGING_STATION CHARGING_STATION_11
12 43 126 CHARGING_STATION CHARGING_STATION_12
13 87 18 CHARGING_STATION CHARGING_STATION_13
14 173 165 CHARGING_STATION CHARGING_STATION_14
15 100 113 CHARGING_STATION CHARGING_STATION_15
16 95 95 CHARGING_STATION CHARGING_STATION_16
17 28 135 CITY CITY_17
18 81 124 CHARGING_STATION CHARGING_STATION_18
19 46 135 RESTAURANT RESTAURANT_19
20 77 1 CHARGING_STATION CHARGING_STATION_20
21 198 84 RESTAURANT RESTAURANT_21
22 120 113 CHARGING_STATION CHARGING_STATION_22
23 10 196 REGULAR_WAYPOINT REGULAR_WAYPOINT_23
24 8 117 CHARGING_STATION CHARGING_STATION_24
25 82 41 REGULAR_WAYPOINT REGULAR_WAYPOINT_25
26 132 11 CHARGING_STATION CHARGING_STATION_26
27 199 120 RESTAURANT RESTAURANT_27
28 162 49 CHARGING_STATION CHARGING_STATION_28
29 200 200 CITY End
