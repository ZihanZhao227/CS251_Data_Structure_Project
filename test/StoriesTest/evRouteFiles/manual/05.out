0 29 60.000 -1.0
NO_PATH
